File: 21.11.04_score&grade.java
import java.io.*;
import java.util.*;
class Main {
	public static char grade(double result){
		char grade2 = 'F';
		if (result>=90){
			grade2 = 'A';
		}else if(result<90 && result>=80){
			grade2 = 'B';
		}else if(result<80 && result>=70){
			grade2 = 'C';
		}else if(result<70 && result>=60){
			grade2 = 'D';
		}
		return grade2;
	}
	
	public static void main(String[] args) throws Exception {
		Scanner sc = new Scanner(System.in);
		String[] in = sc.nextLine().split(" ");
		int sum = (Integer.parseInt(in[0]) + Integer.parseInt(in[1]) + Integer.parseInt(in[2]));
		double result = Math.round(sum / 3.0 * 100)/100.0;
		char grade2 = grade(result);
		System.out.printf("%.2f", result);
		System.out.print(" " + grade2);
		
	}
}

File: 21.11.04_score&grade.py


def grade(result):
	grade2 = 'F'
	if result>=90:
		grade2 = "A"
	elif 80<=result<90:
		grade2 = "B"
	elif 70<=result<80:
		grade2 = "C"
	elif 60<=result<70:
		grade2 = "D"
	return grade2	
	
in1 = list(map(int,input().split(" ")))
sum = in1[0] + in1[1] + in1[2]
result = round(sum/3, 2)
grade2 = grade(result)
print(f'{result:.2f}', grade2)
		
